### PR TITLE
Adding Webservers mod to manifest.json

### PR DESCRIPTION
**Pull request by Zetaphor**
Added Webservers mod

### CODE REVIEW

**Comment by EIA485** [comment]
hey, can you include Swan.Lite as a dependency instead of as an artifact?

**Review by zkxs — CHANGES_REQUESTED** [review]
I believe the way you are combining paths [here](https://github.com/Zetaphor/NeosWebServers/blob/0cde5701b11f86d4de4cc26d815cbc85675bf0ec/Webservers/Webservers.cs#L56) will only work on systems that use backslash as a path separator (e.g. Windows). You should either fix this by using [Path.Combine()](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-6.0), or add `"flags": ["broken:linux-native"]` to your `"1.0.0"` object in the manifest.

EmbedIO appears widely-used and well-maintained, so I don't think we need to worry about it.

I've verified that the EmbedIO and Swan.Lite hashes are correct. Leaving this here for my notes:
```
21392a5fa8d00f4fed9450f7fabd2c8e5dd30cba73525c35f37584e17c639e2b *EmbedIO.dll
309f02bfe8aa4076794e74e95f0610dd03f14c612fce0855df3fe2391e2e990b *Swan.Lite.dll
```

**Comment by zkxs** [comment]
Ok, I'm actually sort of in a mild panic right now because I'm realizing the manifest documentation is trash (I'm allowed to say that because I wrote it) and I've been subjecting people to a Bad Time.

In the interest of getting this merged and not causing you a headache, please just forget about the dependencies thing for now and one of us manifest nerds can deal with it later.

Let me know if you're comfortable dealing with the linux path thing. If not I can also slap that into the manifest myself.

tldr; revert to 0fba394 and I'll merge this.

**Comment by Zetaphor** [comment]
😆 No worries, I'll make another update to move it back. Just published 1.1 to fix the issue for Linux users

**Comment by Zetaphor** [comment]
Should be all set now? Let me know if anything needs improving. I expect to finish implementing the rest of the settings this week

**Review by zkxs — DISMISSED** [review]
Good enough. We can fix any nitpicks we have later.

**Review by zkxs — APPROVED** [review]
